lib/parse.js

```diff
@@ -197,6 +203,12 @@ module.exports = {
 
     // Use files map if present instead of reading files.
     if (!_.isEmpty(files)) {
+
+      // Use pathBrowserify if the origin of input is browser.
+      if (options.browser) {
```

Comment:
@umeshp7 Where are we assigning `browser` as option for this?

Reply:
https://github.com/postmanlabs/openapi-to-postman/pull/377/files#diff-7386b26f8b249e78b58c0ee652f3fca278f4a1d3c277790c8023e7ac6b6a69a7R176 here